### PR TITLE
chore(Portal): simplify Anchor/Link usage

### PR DESCRIPTION
**Pull request by tujoworker**
- Also, disable unwanted scroll to top on page changes.
- Disable the tabs animation.
- Add feature for preloading pages when hovering on tabs or breadcrumb items.

### CODE REVIEW

File: packages/dnb-design-system-portal/src/core/PortalLayout.tsx [hunk 1]
```diff
@@ -136,11 +137,23 @@ export default function PortalLayout(props: PortalLayoutProps) {
...
+          {fmData.breadcrumb.map((item, i, a) => {
+            return (
+              <Breadcrumb.Item
+                key={item.text}
+                variant={
+                  (i == 0 && 'home') ||
+                  (i == a.length - 1 && 'current') ||
+                  null
+                }
+                element={Link}
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/core/PortalLayout.tsx:150]: We merge in the Gatsby Link component, so when hovering, the page will be preloaded.

File: packages/dnb-design-system-portal/src/shared/tags/Tabbar.tsx [hunk 2]
```diff
@@ -114,7 +115,7 @@ export default function Tabbar({
       )}
       <Tabs
         id="tabbar"
-        tab_element={{ ...Link }}
+        tab_element={Link}
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/shared/tags/Tabbar.tsx:118]: Also here, we use the preload capabilities when hovering on a tab.

File: packages/dnb-design-system-portal/gatsby-browser.tsx [hunk 3]
```diff
@@ -64,3 +55,13 @@ export const onRouteUpdate = ({ location, prevLocation }) => {
     applyPageFocus('content')
   }
 }
+
+export const onPreRouteUpdate = ({ prevLocation }) => {
+  if (prevLocation && !globalThis.IS_TEST) {
+    // Opmit scrolling on page changes
```
Comment (langz) [inline packages/dnb-design-system-portal/gatsby-browser.tsx:61]:
```suggestion
    // Omit scrolling on page changes
```

**Review by langz — APPROVED** [review]
Looks great 🎸 

I gave the deploy preview some testing.
I noticed a new error/warning in the console(not sure if it's related to this PR):
Screenshot 2023-09-27 at 10 43 49
Screenshot 2023-09-27 at 10 41 57


Here's a video of an error from https://eufemia-git-chore-simplify-portal-anchor-usage-eufemia.vercel.app/uilib/usage/first-steps/ 


https://github.com/dnbexperience/eufemia/assets/1359205/2cc5c4cb-1c4a-4f39-89f3-cdaa86167dd1

**Pull request merged**